src/sdk/staking.ts

```diff
@@ -197,9 +197,9 @@ async function metaMyStake(web3: Web3, address: string, account: string, isV2: b
     let amount$: CurrencyAmount<Currency>
     if (tokenPrice) {
         const _amountUSDC = amount.multiply(tokenPrice).divide(10 ** token.decimals)
-        amount$ = CurrencyAmount.fromFractionalAmount(USDC[chainId], _amountUSDC.numerator, _amountUSDC.denominator)
+        amount$ = CurrencyAmount.fromFractionalAmount(USDC[SupportedChainId.MAINNET], _amountUSDC.numerator, _amountUSDC.denominator)
     } else {
-        amount$ = CurrencyAmount.fromRawAmount(USDC[chainId], 0)
+        amount$ = CurrencyAmount.fromRawAmount(USDC[SupportedChainId.MAINNET], 0)
```

Comment:
if now you return all the return values as G$Token ie 2 decimals, why are we still returning amount$ as USDC ie 6 decimals?

Reply:
what I believe to understand is that because _amountUSD just above uses USDC as the tokenPrice calculation (through tokenPriceInUSDC)
changing to G$ here would require us to change the tokenPrice calculation right?

this was from point of view right now the fix with the least amount of changes required